src/main/java/org/broadinstitute/hellbender/engine/dataflow/DataflowCommandLineProgram.java

```diff
@@ -25,6 +26,8 @@
 public abstract class DataflowCommandLineProgram extends CommandLineProgram implements Serializable {
     private static final long serialVersionUID = 1l;
 
+    public interface HellbenderDataflowOptions extends GCSOptions, DataflowPipelineOptions {}
```

Comment:
Why do we need this? GCSOptions extends DataflowPipelineOptions (from GenomicsOptions).